integrations/webchat/contributing/template/server/nodejs-express/server.js

```diff
@@ -1,29 +1,32 @@
-#!/usr/bin/env node
-
 /**
- * This file starts up a basic NodeJS server with express, adds some routes, and adds some error handling.
- *
- * ADD A COMMENT HERE ABOUT HOW TO FIND THE SPECIAL CODE YOU HAVE ADDED FOR THIS EXAMPLE.
+ * This file starts up a basic NodeJS server with express, adds some routes, and adds some error handling. This is
+ * mostly boilerplate content that can be adjusted as needed.
  *
  * The server defaults to being available on port 3001.
  */
 
 const { createServer } = require('http');
 
+const cookieParser = require('cookie-parser');
+const cors = require('cors');
```

Comment:
Oh does this mean we don't have to worry about running http-server?

Reply:
Mmmmm, I don't understand. I'm not entirely sure why you had http-server for the basic javascript examples which are html files that can just be opened directly in a browser. It might be necessary if we have a javascript-with-build or react though.

I just added this so you can open an html file locally (or I was doing it through an internal webserver that IJ runs) and not have cors issues trying to make a back-end call. I don't see what http-server would have done to address that. It wouldn't have been on the same port.

Reply:
Ok, ok. I was confused why we needed it!